release-notes-1.2.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+ - [SCMOD-7821] - The converter added for standard logging format to Opensuse-tomcat-image access-log.
```

Comment:
This does not make any sense to me, without any prior knowledge of the change what does this mean? Is this only specific to tomcat? Can this be used elsewhere than our tomcat base image? If so then we shouldn't be calling this out here, this is an independent project to the tomcat one. Can you make this more generic description and not refer to the tomcat image?

Reply:
**Is this only specific to tomcat?**
 - Tomcat specific converter added for threadId and process id pattern. Also, classic logging converter also refactored for Threadid and process id pattern in logging format.
 **Can this be used elsewhere than our tomcat base image?** 
- As of now not. if any access log requirement comes for another web server, this can be used. 
The caf-logging-logback-access-converters module is used by tomcat.

Reply:
**Is this only specific to tomcat?**
 - Tomcat specific converter added for threadId and process id pattern. Also, classic logging converter also refactored for Threadid and process id pattern in logging format.
 **Can this be used elsewhere than our tomcat base image?** 
- As of now not. if any access log requirement comes for another web server, this can be used.